Library/Formula/h264enc.rb

```diff
@@ -0,0 +1,76 @@
+class H264enc < Formula
+  homepage "http://h264enc.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/h264enc/h264enc/h264enc-9.9.0.tar.gz"
+  sha1 "8bfa7017386a4b7612acd21e48dedf940652582d"
+
+  # depends_on "cmake" => :build
+  # depends_on :x11 # if your formula requires any X11/XQuartz components
+  # [encode, x264], mplayer needs x264 installed first 
```

Comment:
Just to check: are you specifying these dependencies so that mplayer will link against them?